app/grants/templates/grants/detail/template-grant-details.html

```diff
@@ -430,6 +427,18 @@ <h5 class="font-weight-bold mb-3">Verify Grant Ownership with Twitter</h5>
           </div>
         </div>
 
+        <!-- Funding Info -->
+        <div class="col-12 mb-3">
+          <label class="font-caption letter-spacing text-black-60 text-uppercase" for="funding_info">Project Funding Info</label>
+          <span class="font-smaller-6 badge badge-greylight text-capitalize ml-2">Required</span>
+
+          <input id="funding_info" v-model="grant.funding_info" name="funding_info" class="form__input form__input-lg" type="text" placeholder="Raised $1m from VC.Fund" required>
```

Comment:
If this info isn't required should we drop the required attr and badge (from here and from `_new.html`)?